.github/workflows/daily.yml

```diff
@@ -379,10 +379,10 @@ jobs:
         run: sudo apt-get install tcl8.6 tclx
       - name: test
         if: true && !contains(github.event.inputs.skiptests, 'valkey')
-        run: ./runtest --config io-threads 2 --config events-per-io-thread 0 --accurate --verbose --tags network --dump-logs ${{github.event.inputs.test_args}}
+        run: ./runtest --io-threads --accurate --verbose --tags network --dump-logs ${{github.event.inputs.test_args}}
       - name: cluster tests
         if: true && !contains(github.event.inputs.skiptests, 'cluster')
-        run: ./runtest-cluster --config io-threads 2 --config events-per-io-thread 0 ${{github.event.inputs.cluster_test_args}}
+        run: ./runtest-cluster --io-threads ${{github.event.inputs.cluster_test_args}}
```

Comment:
runtest-cluster does not support --io-threads option, they are different framework. On the other hands, runtest-cluster doesnot support tags. And on the other hands, i see we don't have io-threads test in cluster-tests (and we dit not change anything in cluster tests), so this is a mistake?

Unknown option --io-threads

Reply:
#933